app/src/features/rules/constants/index.ts

```diff
@@ -1,3 +1,9 @@
 import { RuleType } from "types";
 
 export const PREMIUM_RULE_TYPES = [RuleType.RESPONSE, RuleType.REQUEST, RuleType.SCRIPT];
+
+export const RULE_IMPORT_TYPE = {
+  OVERWRITE: "overwrite",
+  DUPLICATE: "duplicate",
+  NORMAL: "normal",
+};
```

Comment:
If default is overwrite then can we remove `normal`. Lets keep only 2 tyes - `overwrite` and `duplicate`